solarforecastarbiter/metrics/calculator.py

```diff
@@ -243,15 +257,13 @@ def calculate_probabilistic_metrics(processed_fx_obs, categories, metrics,
     ref_fx_obs : datamodel.ProcessedForecastObservation
         Reference forecast to be used when calculating skill metrics. Default
         is None and no skill metrics will be calculated.
-
```

Comment:
I think numpydoc format is blank lines between sections https://numpydoc.readthedocs.io/en/latest/example.html#example

Reply:
Correct. The line removal was a mistake I made when resolving a git rebase conflict. I'll fix it now.